src/averageAges.js

```diff
@@ -37,7 +40,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people.filter(person =>
+    person.sex === 'f'
+    && (withChildren === undefined
```

Comment:
Instead of checking for 'withChildren === undefined', use '!withChildren' for better readability.